napari/utils/settings/_defaults.py

```diff
@@ -0,0 +1,89 @@
+"""Settings management.
+"""
+
+from enum import Enum
+from typing import List, Tuple
+
+from pydantic import BaseSettings, Field
+
+from napari.utils.events.evented_model import EventedModel
+
+
+class QtBindingEnum(str, Enum):
+    """Python Qt binding to use with the application."""
+
+    pyside = 'pyside2'
+    pyqt = 'pyqt5'
+
+
+class ThemeEnum(str, Enum):
+    """Application color theme."""
+
+    dark = 'dark'
+    light = 'light'
+
+
+class ApplicationSettings(BaseSettings, EventedModel):
+    """Main application settings."""
+
+    version = (0, 1, 0)
+    # Python
+    qt_binding: QtBindingEnum = QtBindingEnum.pyside
```

Comment:
happy to be overridden here, but I think it's better for the choices to be pyqt5 and pyside2, as those are the strings (?) that QtPy expects from QT_API? And based on the str values for the Enum I guess I'm correct with that. I find it confusing and annoying for this to be inconsistent e.g. in conda-forge.

Reply:
i'd probably stick with `pyside2` and `pyqt5` too, but agree it's all a little confusing. The next releases will be `pyqt6` and `pyside6` for `qt6`. `pysid3`, `pysid4`, and `pysid5` will never exist :-)

Reply:
Fixing